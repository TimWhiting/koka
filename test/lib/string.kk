// See https://github.com/koka-lang/koka/issues/486
fun main()
  val some-string = "hello"
  println("some-string = [" ++ some-string ++ "]")
  println("some-string.vector.string = [" ++ some-string.vector.string ++ "]")
