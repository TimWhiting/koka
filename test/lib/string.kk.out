some-string = [hello]
some-string@vector@string = [hello]